GUI/MainModList.cs

```diff
@@ -204,8 +204,7 @@ private void _UpdateModsList(IEnumerable<ModChange> mc, Dictionary<string, bool>
             {
                 foreach (GUIMod gm in gui_mods)
                 {
-                    bool oldIncompat;
-                    if (old_modules.TryGetValue(gm.Identifier, out oldIncompat))
+                    if (old_modules.TryGetValue(gm.Identifier, out bool oldIncompat))
```

Comment:
Ooh, I like that `out bool` syntax! I can think of many uses for that.
(I think this is the second time I've learned something useful about `out` params from reviewing your code? :1st_place_medal:)